src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -101,6 +101,33 @@ def stop(self, wf_run_id: WorkflowRunId, config_name: ConfigName):
             # Other exception types aren't expected to be raised here.
             raise
 
+    def get_wf_outputs(self, wf_run_id: WorkflowRunId, config_name: ConfigName):
+        """
+        Asks the runtime for workflow output values.
+
+        If the workflow is still executing this will block until the workflow
+        completion.
+
+        Raises:
+            orquestra.sdk.exceptions.NotFoundError: when the run_id doesn't match a
+                stored run ID.
+            orquestra.sdk.exceptions.ConfigNameNotFoundError: when the named config is
+                not found in the file.
+            orquestra.sdk.exceptions.WorkflowRunNotSucceeded: when the workflow is no
+                longer executing, but wasn't succeded.
+        """
+        try:
+            wf_run = sdk.WorkflowRun.by_id(wf_run_id, config_name)
+        except (exceptions.NotFoundError, exceptions.ConfigNameNotFoundError):
+            raise
+
+        try:
+            outputs = wf_run.get_results(wait=True)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Do we always want this to block? This may look like a broken CLI if we don't show anything

Reply:
Raising an error (which `get_results` does) should be enough?

Reply:
Changed in 19d53fc